cmd/meroxa/turbine_cli/utils.go

```diff
@@ -237,11 +238,15 @@ func GetGitSha(appPath string) (string, error) {
 	return string(output), nil
 }
 
-func GoInit(l log.Logger, appPath string, skipInit, vendor bool) error {
+func GoInit(ctx context.Context, l log.Logger, appPath string, skipInit, vendor bool) error {
 	l.StartSpinner("\t", "Running golang module initializing...")
 	skipLog := "skipping go module initialization\n\tFor guidance, visit " +
 		"https://docs.meroxa.com/beta-overview#go-mod-init-for-a-new-golang-turbine-data-application"
 	goPath := os.Getenv("GOPATH")
+	if goPath == "" {
+		goPath = build.Default.GOPATH
+		l.Infof(ctx, "using default GOPATH: %s", goPath)
+	}
 	if goPath == "" {
```

Comment:
@nassor looks like this conditional will never take place (if `goPath` was empty, then it was set to `build.Default.GOPATH`.  We should remove this one also stopping the spinner since we started it on[ line 242](https://github.com/meroxa/cli/pull/408/files#diff-26b7e43c725b91b24ee83f56a2e8a9f0e595b868fb1d50d33c9bff65592a15b9R242).

Reply:
@nassor ignore me. I just saw your follow up PR https://github.com/meroxa/cli/pull/409 👍